msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationJavaTest.java

```diff
@@ -2832,6 +2974,38 @@ public void testSignUpInvalidEmailReturnsError() throws ExecutionException, Inte
 
         assertTrue(signUpResult instanceof SignUpError);
         assertTrue(((SignUpError) signUpResult).isInvalidUsername());
+
+        checkSafeLogging();
+    }
+
+    private void checkSafeLogging() {
```

Comment:
Since this is used in both tests, and potentially more tests moving forward: can this be extracted into a dedicated component, so that reusability becomes more straight forward?

Reply:
Create a component called as LoggerCheckHelper which has all logger related stuff inside.